README.md

```diff
@@ -11,7 +11,7 @@ Navigate to the
 Contribution docs are part of the docs site linked above. Some direct
 links:
 
-* [Contributing to documentation](https://openstack-k8s-operators.github.io/data-plane-adoption/contributing/documentation/) -
+* [Contributing to documentation](https://openstack-k8s-operators.github.io/data-plane-adoption#_contributing_to_documentation) -
```

Comment:
Still broken. This one works: `https://openstack-k8s-operators.github.io/data-plane-adoption/dev/#_contributing_to_documentation`

Reply:
Good catch. I've set to correct URL in next PR